.gitignore

```diff
@@ -3,3 +3,4 @@
 *.rs.rustfmt
 target/
 Cargo.lock
+.idea/
```

Comment:
Actually, I just found out I can edit `../.git/info/exclude` for project specific non-committed excluding. 🎉